### PR TITLE
Deploy shared Azure resources

### LINKED ISSUE #6: Deploy the sample application

### PR DESCRIPTION
**Pull request by CS-JAA**
closes #6

### CODE REVIEW

File: .github/workflows/day4-scm-common.yml [hunk 1]
```diff
@@ -21,7 +21,8 @@ env:
 
 jobs:
   build:
-    if: github.repository == 'azuredevcollege/trainingdays'
+    if: github.repository == 'day4-challenge2
```
Comment (zidan95) [inline .github/workflows/day4-scm-common.yml:24]: please remove linebreak

File: .github/workflows/day4-scm-common.yml [hunk 2]
```diff
@@ -45,7 +46,8 @@ jobs:
           retention-days: 30
 
   deploy-to-dev:
-    if: (github.repository == 'azuredevcollege/trainingdays') && ((github.event_name == 'push') || (github.event_name == 'workflow_dispatch'))
+    if: (github.repository == 'day4-challenge2
```
Comment (zidan95) [inline .github/workflows/day4-scm-common.yml:49]: please remove linebreak

File: .github/workflows/day4-scm-common.yml [hunk 3]
```diff
@@ -87,7 +89,8 @@ jobs:
           parameters: env=${{ env.ENV_NAME }}
 
   deploy-to-test:
-    if: (github.repository == 'azuredevcollege/trainingdays') && ((github.event_name == 'push') || (github.event_name == 'workflow_dispatch'))
+    if: (github.repository == 'day4-challenge2
```
Comment (zidan95) [inline .github/workflows/day4-scm-common.yml:92]: also here

**Review by zidan95 — CHANGES_REQUESTED** [review]
please remove linebreak

File: .github/workflows/day4-scm-common.yml (hunk 1 above)
Comment (CS-JAA) [reply .github/workflows/day4-scm-common.yml:24]: will do 👍🏻

**Comment by CS-JAA** [comment]
[Added in this commit](https://github.com/azure-college-jja/trainingdays/commit/fe517e2ef862f87e7fcc3991be9dee0b452d437c)

**Review by zidan95 — APPROVED** [review]

**Pull request merged**